.circleci/run_tests.sh

```diff
@@ -31,6 +31,16 @@ sleep 1
 # Create a database at port 5433, forward it to Postgres
 toxiproxy-cli create -l 127.0.0.1:5433 -u 127.0.0.1:5432 postgres_replica
 
+#
+# Ruby integration tests
+# These tests create their own PgCat servers so we want to run them after starting toxiproxy
+# and before starting PgCat
+#
+cd tests/ruby
+sudo bundle install
+bundle exec rspec *_spec.rb --format documentation || exit 1
+cd ../..
+
```

Comment:
I am wondering if the fact that we have an extra PgCat running in the background is making the specs flake out. So I moved the ruby tests up before we start any PgCats

Reply:
Nope :(